central/notifiers/cscc/cscc.go

```diff
@@ -115,7 +114,10 @@ func (c *cscc) AlertNotify(ctx context.Context, alert *storage.Alert) error {
 		return err
 	}
 
-	_, err = c.client.CreateFinding(ctx, &securitycenterpb.CreateFindingRequest{
+	client, done := c.client.GetClient()
```

Comment:
I think you have to do a nil check here, because the client cannot be guaranteed to exist. For example if this is called before the credential fetch ever succeeded.